R/model-tree.R

```diff
@@ -381,14 +381,25 @@ make_model_network <- function(full_log){
 
   # Create Network
   start <- "Start"
-  noref <- data.frame(from = NA, to = start)
-  parent_mods <- data.frame(from = start,
-                            to = full_log$run[full_log$based_on==""])
-  child_mods <- data.frame(from = full_log$based_on[full_log$based_on!=""],
-                           to = full_log$run[full_log$based_on!=""])
-  network_df <- rbind(noref, parent_mods, child_mods)
-
-  # Check network links - set unlinked models as starting points
+  noref_df <- data.frame(from = NA, to = start)
+
+  child_mods_df <- data.frame(
+    from = full_log$based_on[full_log$based_on!=""],
+    to = full_log$run[full_log$based_on!=""]
+  )
+
+  parent_mods <- full_log$run[full_log$based_on==""]
+  if(!rlang::is_empty(parent_mods)){
```

Comment:
nit-pick: if you negate the condition of an if/else, it is almost always more readable (to my eyes) to drop the negation (i.e. remove `!` and flip the if/else arms

Reply:
Fair. I like to have the more likely scenario appear first but I see the argument for that